test_httpbin/steps/request_steps_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Req.Steps.RequestStepsTest do
+  use ExUnit.Case, async: true
+  #!TODO: make `run_steps/2` testabable
+  doctest Req.Steps,
+    only: [
+      put_base_url: 2,
+      encode_headers: 1,
+      encode_body: 1,
+      put_params: 2,
+      put_range: 2,
+      put_if_modified_since: 2
+    ]
+end
```

Comment:
Could you do the following changes?

1. Move all `doctest` declarations to a single case: `test/httpbin_test.exs`
2. Add `@moduletag :httpbin` to it
3. in test/helper run `unless System.get_env("CI") do ExUnit.configure(exclude: :httpbin) end`

WDYT?

Reply:
regarding 1. I personally like more separation of different tests but if you prefer a single testfile I can update that this way.

Reply:
Let's start with a single test case and see how it goes, we can always break it apart down the road if it's weird.